ansible/roles/enomoloch-get-pcaps/files/get_messages.sh

```diff
@@ -0,0 +1,7 @@
+while true; do
+ 
+rsync -WSat --remove-source-files --include='*.pcap' root@engine:/pcaps/data/ /pcaps/;
```

Comment:
You should test whether this works as intended. I had problems in the past with Moloch not recognizing files added through rsync. What worked for me was pushing the files via scp from the source to the destination

Reply:
Any update here?